ios-trivia-game/ViewControllers/CameraViewController.swift

```diff
@@ -10,6 +10,14 @@ import UIKit
 
 class CameraViewController: UIViewController {
 
+    @IBAction func onBack(_ sender: UIBarButtonItem) {
+        let storyboard = UIStoryboard(name: "Main", bundle: nil)
+        let destination = storyboard.instantiateViewController(withIdentifier: "com.iostriviagame.maintabviewcontroller") as! MainTabViewController
+        destination.selectedIndex = 2
+        destination.navigationController?.isNavigationBarHidden = true
```

Comment:
this is how i got the back button to stop displaying when i navigated back to to the profile view controller in the tabviewcontroller.

by default tabviewcontroller has its own navigationcontroller